charts/kube-prometheus-stack/crds/crd-alertmanagers.yaml

```diff
@@ -973,6 +973,20 @@ spec:
                       are ANDed.
                     type: object
                 type: object
+              alertmanagerConfiguration:
+                description: 'EXPERIMENTAL: alertmanagerConfiguration specifies the
```

Comment:
@mrueg, would you know if this feature is still `EXPERIMENTAL`? It says so on the [main branch currently](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/charts/crds/crds/crd-alertmanagers.yaml#L1142-L1143).